### DISCUSSION TITLE
Integration, external data, etc.

### DISCUSSION THREAD

**Discussion by casals** [anyproto/contributors#18] [Q&A] [answered]
Are there any plans for allowing integration with external apps/APIs/DBs (RT-synced or not)/triggers/hooks/etc? (Or- if it's something I could do today - could you point me to the right direction?)

**Comment by fuksman** [comment, marked as answer]
@casals yes, for sure! I've described approximate plans [here](https://github.com/orgs/anyproto/discussions/9), Track 3.
Works are planned for this and next seasons, I hope we will be able to provide the first version of API for integrations in one-two months.

**Reply by casals** [reply]
Thanks! If you need devs as alpha testers please count on me :)

**Reply by fuksman** [reply]
Deal 🤝

**Reply by reecenward** [reply]
is this out now?

**Reply by fuksman** [reply]
Hey @reecenward, we've published `any-block` protocol and suggested creating data converters based on this (please see https://github.com/orgs/anyproto/discussions/82 for details). It's not a proper API yet, but it already allows creating some tools for data transfer to/from Anyytpe.
